packages/next-swc/crates/napi/src/next_api/project.rs

```diff
@@ -876,7 +878,10 @@ pub async fn project_trace_source(
             };
 
             let token = map
-                .lookup_token(frame.line as usize, frame.column.unwrap_or(0) as usize)
+                .lookup_token(
+                    frame.line as usize - 1,
+                    frame.column.unwrap_or(1) as usize - 1,
```

Comment:
This might cause an underflow

```suggestion
                    (frame.line as usize).saturating_sub(1),
                    (frame.column.unwrap_or(1) as usize).saturating_sub(1),
```